src/spdx_tools/spdx/model/relationship.py

```diff
@@ -73,3 +73,6 @@ def __init__(
         comment: Optional[str] = None,
     ):
         check_types_and_set_values(self, locals())
+
+    def __hash__(self):
```

Comment:
this is not a correct hash of the relationship. One might instead return the triple and use it for lookups in the cache